From a6ca3c289994a3a9ab8f481582e2632109b759f0 Mon Sep 17 00:00:00 2001
From: Egmont Koblinger <egmont@gmail.com>
Date: Thu, 19 Jan 2012 14:11:10 +0400
Subject: [PATCH] Ticket #2689 (fix fallocate detection for xBSD)

    fixed macros 'mc-get-fs-info.m4' for correct 'fallocate' detection under xBSD

Signed-off-by: Ilia Maslakov <il.smind@gmail.com>
---
 m4.include/mc-get-fs-info.m4 |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/m4.include/mc-get-fs-info.m4 b/m4.include/mc-get-fs-info.m4
index bc0ca54..b1474d8 100644
--- a/m4.include/mc-get-fs-info.m4
+++ b/m4.include/mc-get-fs-info.m4
@@ -5,7 +5,7 @@ dnl
 
 AC_DEFUN([gl_POSIX_FALLOCATE], [
     dnl * Old glibcs have broken posix_fallocate(). Make sure not to use it.
-    AC_TRY_COMPILE([
+    AC_TRY_LINK([
         #define _XOPEN_SOURCE 600
         #include <stdlib.h>
         #if defined(__GLIBC__) && (__GLIBC__ < 2 || __GLIBC_MINOR__ < 7)
-- 
1.7.8.3

